pyacq/core/rpc.py

```diff
@@ -240,7 +268,23 @@ def __init__(self, client, method):
         self.method = method
         
     def __call__(self, *args, **kwds):
-        return self.client._call_method(self.method, *args, **kwds)
+        # This little hack could be a help for manger remote method when dealing with node
+        # for instance:
+        # nodename = mynodeproxy.name
+        # manager.create_node_outputs(nodename, streamdef)
+        # could be directly be
+        # manager.create_node_outputs(mynodeproxy, streamdef)
```

Comment:
This is moving toward having proper proxy objects for all RPC interacion, which I think would be generally useful.
In pyqtgraph's multiprocess system, each remote call can either serialize and send back its return value, or it can keep a local reference to the return value and send back a new proxy object. This allows for a lot of flexibility in the way you control remote processes, particularly in that you can then feed those return values back into further remote calls without ever having transferred the data itself. For example:

```
# generate and process an array remotely without ever transferring it to the calling process
array_proxy = node_proxy.get_array()
node_proxy.process_data(array_proxy)
```